File: client/src/app/modules/home/features/member/models/message.ts
export interface Message {
    id: number;
    SenderUserId: number;
    RecipientUserId: number;
    senderPhotoUrl: string;
    content: string;
    dateRead?: Date;
    messageSent: Date;
    senderDeleted: boolean,
    recipientDeleted: boolean,
}

File: API/Entities/Message.cs
namespace API.Entities
{
    public class Message
    {
        public int Id { get; set; }
        public int SenderUserId { get; set; }
        public int RecipientUserId { get; set; }
        public string Content { get; set; }
        public DateTime? DateRead { get; set; }
        public DateTime MessageSent { get; set; } = DateTime.UtcNow;
        public bool SenderDeleted { get; set; }
        public bool RecipientDeleted { get; set; }
    }
}